src/neoxp/Node/OfflineNode.cs

```diff
@@ -406,6 +408,14 @@ IReadOnlyList<ExpressStorage> ListStorages(UInt160 scriptHash)
 
         public Task<IReadOnlyList<ExpressStorage>> ListStoragesAsync(UInt160 scriptHash)
             => MakeAsync(() => ListStorages(scriptHash));
+        
+        int PersistContract(ContractState state, (string key, string value)[] storagePairs, ContractCommand.ContractForce? force)
```

Comment:
There's no need to have the PersistContract method since it's just a one liner. Just put the call to NodeUtility.PersistContract inside the MakeAsync call of PersistContractAsync

Reply:
I thought we always wanted to have a `sync` version as well, but happy to remove